panoramapublic/test/src/org/labkey/test/tests/panoramapublic/PanoramaWebPublicSearchTest.java

```diff
@@ -157,6 +157,7 @@ public void testProteinSearch()
         checker().verifyEquals("Incorrect protein searched with partial match", 0, table.getDataRowCount());
 
         log("Protein : Partial match and results across folder");
+        clickAndWait(Locator.id("clear-all-button-id-protein"));
         panoramaPublicSearch = new PanoramaPublicSearchWebPart(getDriver(), "Panorama Public Search");
         table = panoramaPublicSearch.gotoProteinSearch().setProtein("R").search();
```

Comment:
Adding clearAll method in PanoramaPublicSearchWebPart would be better

final WebElement clearAll = Locator.button("Clear All").findWhenNeeded(this); // in ElementCache

public PanoramaPublicSearchWebPart clearAll()
    {
        getWrapper().clickAndWait(elementCache().clearAll);
        return this;
    }

Reply:
Using Locator.button("Clear All") did not work for me before, hence the usage of "Locator.id("clear-all-button-id-protein")"

Anyway, adding the above code in PanoramaPublicSearchWebPart gives me this:
 Driver info: org.openqa.selenium.chrome.ChromeDriver
    Command: [bd6ab9cb9c22ce73486a0f8844a18aff, clickElement {id=a21b5911-2a11-4230-ba3f-f4d2a2bda69c}]
    Capabilities {acceptInsecureCerts: false, browserName: chrome, browserVersion: 109.0.5414.120, chrome: {chromedriverVersion: 108.0.5359.71 (1e0e3868ee06..., userDataDir: C:\Users\BinalP\AppData\Loc...}, goog:chromeOptions: {debuggerAddress: loc
alhost:62648}, javascriptEnabled: true, networkConnectionEnabled: false, pageLoadStrategy: normal, platform: WINDOWS, platformName: WINDOWS, proxy: Proxy(), se:cdp: ws://localhost:62648/devtoo..., se:cdpVersion: 109.0.5414.120, setWindowRect: true,
 strictFileInteractability: false, timeouts: {implicit: 0, pageLoad: 300000, script: 30000}, unhandledPromptBehavior: dismiss and notify, webauthn:extension:credBlob: true, webauthn:extension:largeBlob: true, webauthn:virtualAuthenticators: true}
    Element: [[[[ChromeDriver: chrome on WINDOWS (bd6ab9cb9c22ce73486a0f8844a18aff)] -> xpath: (//div[@name="webpart"][div/div/*[contains(concat(' ',normalize-space(@class),' '), " panel-title ")][@title="Panorama Public Search"]])[1]]] -> xpath: .
//button[not(ancestor-or-self::*[contains(@style,'display: none') or contains(@style,'visibility: hidden') or contains(@class, 'x-hide-display') or contains(@class, 'x4-hide-offsets') or contains(@class, 'x-hide-offsets')] or (@type = 'hidden'))][n
ot(contains(@class, 'tab'))][normalize-space()="Clear All"]]
    Session ID: bd6ab9cb9c22ce73486a0f8844a18aff
        at java.base/jdk.internal.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
        at java.base/jdk.internal.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:77)
        at java.base/jdk.internal.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
        at java.base/java.lang.reflect.Constructor.newInstanceWithCaller(Constructor.java:499)
        at java.base/java.lang.reflect.Constructor.newInstance(Constructor.java:480)
        at org.openqa.selenium.remote.codec.w3c.W3CHttpResponseCodec.createException(W3CHttpResponseCodec.java:200)
        at org.openqa.selenium.remote.codec.w3c.W3CHttpResponseCodec.decode(W3CHttpResponseCodec.java:133)
        at org.openqa.selenium.remote.codec.w3c.W3CHttpResponseCodec.decode(W3CHttpResponseCodec.java:53)
        at org.openqa.selenium.remote.HttpCommandExecutor.execute(HttpCommandExecutor.java:184)
        at org.openqa.selenium.remote.service.DriverCommandExecutor.invokeExecute(DriverCommandExecutor.java:167)
        at org.openqa.selenium.remote.service.DriverCommandExecutor.execute(DriverCommandExecutor.java:142)
        at org.openqa.selenium.remote.RemoteWebDriver.execute(RemoteWebDriver.java:567)
        at org.openqa.selenium.remote.RemoteWebElement.execute(RemoteWebElement.java:251)
        at org.openqa.selenium.remote.RemoteWebElement.click(RemoteWebElement.java:77)
        at org.labkey.test.selenium.WebElementWrapper.click(WebElementWrapper.java:37)
        at org.labkey.test.selenium.ReclickingWebElement.click(ReclickingWebElement.java:66)
        at org.labkey.test.selenium.WebElementWrapper.click(WebElementWrapper.java:37)
        at org.labkey.test.WebDriverWrapper.lambda$52(WebDriverWrapper.java:2761)
        at org.labkey.test.WebDriverWrapper.lambda$22(WebDriverWrapper.java:1925)
        at org.labkey.test.WebDriverWrapper.doAndMaybeWaitForPageToLoad(WebDriverWrapper.java:1953)
        at org.labkey.test.WebDriverWrapper.doAndWaitForPageToLoad(WebDriverWrapper.java:1926)
        at org.labkey.test.WebDriverWrapper.clickAndWait(WebDriverWrapper.java:2759)
        at org.labkey.test.WebDriverWrapper.clickAndWait(WebDriverWrapper.java:2754)
        at org.labkey.test.components.panoramapublic.PanoramaPublicSearchWebPart.clearAll(PanoramaPublicSearchWebPart.java:135)
        at org.labkey.test.tests.panoramapublic.PanoramaWebPublicSearchTest.testPeptideSearch(PanoramaWebPublicSearchTest.java:213)